### ISSUE TITLE
The igGrid table's contents are announced in one go when focused

### ISSUE THREAD

**Issue by kchanda24** [IgniteUI/ignite-ui#2251]
During a recent evaluation by a third-party accessibility vendor, we found out that when navigation through the containing page and upon reaching the datagrid, the entire content of the datagrid is announced in one go by the screen reader. This is not aligned with WCAG 2.0 Level A guidelines. The corresponding WCAG success criteria is 1.3.1 - Info and Relationships.

##Version and Browser used : 

- ignite-ui version: any
- browser: any

## Steps to reproduce : 

Go to this [sample](https://jsfiddle.net/x9qc6hLy/).

Use NVDA or other accessibility tool.
Reach the data table

## Result : 

The screen reader reads the entire content of the data table in one go.

## Expected Result : 

The screen reader should avoid the announcement of the entire content in one go, it will confuse the user.

This additional tab including with entire content announcement should be removed.

**Comment by kchanda24** [comment]
Hi @RivaIvanova! Thanks for looking into this. I understand what you are saying. But the core accessibility issue remains as the tabIndex causes assistive technologies to read out all the content under the grid container, in this case is the full data content of igGrid. 

I would suggest to remove the tabIndex and look for alternate ways of providing the custom logic ensuring that assistive technologies like NVDA, does not read out the whole datatable or hiding the container from assistive technologies so that they do not read the entire content.

**Comment by ChronosSF** [comment]
@kchanda24 , generally speaking tabindex should be applied to interactable elements only and as such the grid container, which being a <div> makes NVDA go crazy readying everything under it, shouldn't really be focusable. It can have certain uses, especially if you want to modify the page tab sequence as we allow you to set it to a custom value through an option, but it's probably doing more harm than good and it's also probably better to have it opt-in than opt-out. I'll look into it a bit more before removing it but in any case, we'll get to it before the end of August.
